website/content/configuration/k3s.md

```diff
@@ -3,7 +3,63 @@ title: Issues with K3s
 weight: 20
 ---
 
-K3S come with its own service load balancer named Klipper. You need to disable it in order to run MetalLB.
-To disable Klipper, run the server with the `--disable servicelb` option, as described in [K3s documentation](https://rancher.com/docs/k3s/latest/en/networking/
-)
+## Conflicts with K3s' own LoadBalancer implementation
 
+K3s come with its own service load balancer named Klipper. You need to disable it in order to run MetalLB.
+To disable Klipper, run the server with the `--disable servicelb` option, as described in [K3s documentation](https://rancher.com/docs/k3s/latest/en/networking/).
+
+As an alternative to disabling Klipper-LB you can configure MetalLB to only recognizes services that have been configured with a specific `loadBalancerClass` attribute,
+as described by the [installation documentation](https://metallb.universe.tf/installation/#setting-the-loadbalancer-class).
+
+For example, if you run the MetalLB `controller` and `speaker` pods with `--lb-class=metallb.universe.tf/metallb-class`, you can then directly reference this class in your services:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-metallb
+  annotations:
+    metallb.universe.tf/address-pool: ingress-ip-pool
+spec:
+  type: LoadBalancer
+  loadBalancerClass: metallb.universe.tf/metallb-class
+  ports:
+  - port: 443
+    targetPort: 8443
+```
+
+Klipper-LB will ignore this service due to its configured `loadBalancerClass`.
```

Comment:
I feel like this is a bit redundant (with what loadBalancerClasses are), but this might belong better to the https://metallb.universe.tf/installation/#setting-the-loadbalancer-class section (as nothing here is Klipper-LB specific) if it helps people. wdyt?

Reply:
Agreed. I've moved the example to the loadBalancerClass section of the installation docs.